src/test/tv/codely/scala_http_api/course/infrastructure/marshaller/CourseMarshaller.scala

```diff
@@ -0,0 +1,17 @@
+package tv.codely.scala_http_api.course.infrastructure.marshaller
+
+import spray.json.{JsArray, JsObject, JsString}
+import tv.codely.scala_http_api.course.domain.Course
+
+object CourseMarshaller {
```

Comment:
In order to maintain consistency between all this kind of marshallers, the naming should follow the one already used in the `VideoJsValueMarshaller` and `UserJsValueMarshaller`, that is: `CourseJsValueMarshaller`.

This way we would be specifying which kind of marshaller it is (`JsValue`) so we're being more explicit and allowing other kinds of marshallers.